tests/sample/ebpf.h

```diff
@@ -69,6 +74,27 @@ typedef struct _IPV4_HEADER
     uint32_t DestinationAddress;
 } IPV4_HEADER, *PIPV4_HEADER;
 
+typedef uint8_t ipv6_address_t[16];
+
+typedef struct _IPV6_HEADER
```

Comment:
Same comment here, this (and IPV4_HEADER) can be removed if we just `#include <netiodef.h>`

Reply:
same as above.